runelite-client/src/main/java/net/runelite/client/plugins/dpscounter/DpsMember.java

```diff
@@ -90,8 +96,11 @@ void reset()
 	{
 		damage = 0;
 		start = end = Instant.now();
+		reset = true;
 	}
 
+	boolean isReset() { return reset; }
```

Comment:
This class is already annotated `@Getter`, you don't need to create an explicit getter.

Reply:
this can just be `Objects.equals(start, end)`